### PR TITLE
Apptype is optional

### PR DESCRIPTION
**Pull request by wmathurin**
It default to native_swift for forceios (and native for forcedroid, hybrid_local for forcehybrid).
Updated help texts and README's accordingly.

### CODE REVIEW

File: android/README.md [hunk 1]
```diff
@@ -71,9 +71,9 @@ Once the creation script completes, you'll have a fully functioning basic applic
 
 ### forcedroid create options
 
-**App Type:** The type of application you wish to develop:
+**App Type:** \( *Optional* \) The type of application you wish to develop:
 
-- **native** — A fully native Android application
+- **native** (default) — A fully native Android application
```
[the remark this answers is not in the document]
Comment (wmathurin) [reply android/README.md]: Is Google pushing for Kotlin the way Apple is pushing for Swift?

[the remark this answers is not in the document]
Comment (bhariharan) [reply android/README.md]: Maybe not at the same level, but the code docs are starting to be `Kotlin` first. Our new project is all `Kotlin` too.

**Comment by rwhitley** [comment]
Is this change for 6.3 or 7.0?

**Comment by wmathurin** [comment]
@rwhitley I don't think we will have a 6.3.

**Pull request merged**